springerlink.md

```diff
@@ -1083,8 +1043,10 @@ Third party advertising scripts may inject images containing images of text.
 <em>Revised Section 508 - does not apply</em>
 </td>
 <td>Partially Supports</td>
-<td>Most parts of this product are responsive. Content can be presented without loss of information or functionality, and without requiring scrolling in two dimensions per the specifications in this criterion.
+<td>Most parts of this product are responsive. Content can be presented without loss of information or functionality, and without requiring scrolling in two dimensions, except where necessary.
```

Comment:
I can't explain well why, but the expression "where necessary" leaves me dissatisfied. What does that mean or who decides whether it is "necessary"? Personally, I'd prefer something more tangible, like for example

```suggestion
<td>Most parts of this product are responsive. Content can be presented without loss of information or functionality, and without requiring scrolling in two dimensions, except where we could not come up with a better solution.
```

Reply:
You're dissatisfied by it because it's a weak statement on its own 😄 The exceptions are in the next sentence, but there was a line break between the two sentences, and I did have a slightly uneasy feeling about separating them like this. I've brought the second sentence up so it lives in the same paragraph as this one!

Reply:
Ah, even better so. Thanks a lot!